src/data/user-charts.js

```diff
@@ -0,0 +1,150 @@
+
+import Chart from 'chart.js/auto'; 
+
+const sleepChart = document.getElementById('sleep-chart');
+const hydrationChart = document.getElementById('hydration-chart');
+const stepsChart = document.getElementById('steps-chart');
+const friendsChart = document.getElementById('friends-avarage-chart');
+// I think this one maybe let not const.
+const friendlabels = ['friend1', 'friend2', 'friend3','friend4','friend5'];
+
+console.log("blabvlabalba");
```

Comment:
Awesome console message! Not sure if this was left in be accident or we will use this later.

Reply:
Good catch! I'm still working in this file, thank you, I will delete the console.log

Reply:
Deleted!